src/Symfony/Component/String/ByteString.php

```diff
@@ -25,20 +25,58 @@
  */
 class ByteString extends AbstractString
 {
+    private const ALPHABET_ALPHANUMERIC = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+
     public function __construct(string $string = '')
     {
         $this->string = $string;
     }
 
-    public static function fromRandom(int $length = 16): self
+    public static function fromRandom(int $length = 16, string $alphabet = null): self
     {
-        $string = '';
+        if (0 === $length) {
+            return new static('');
+        }
+
+        if ($length < 0) {
```

Comment:
let's turn this into an `<=` and remove the previous if

Reply:
that would mean `ByteString::fromRandom(0)` results in exception? :thinking:

Reply:
Yes :)